### ISSUE TITLE
[BUG] Can't get the modpack to run on MacOS Sonoma 14.0

### ISSUE THREAD

**Issue by AR-E** [TCreopargh/GreedyCraft#271]
**Before reporting, make sure your problem isn't caused by the game running out of RAM and you have read the installation guide carefully.**
If the game crashes to your desktop without showing a crash screen it's a RAM issue. Don't submit this kind of issue. Please view the installation guide for how to solve this issue. 
If this is not the case check the checkbox below by replacing the space between brackets with an 'x'.
 - [x] Yes, I'm sure this problem isn't because my game ran out of RAM.

**Describe the bug**
The modpack finishes loading, I hear the ding, and then it crashes with exitcode 6; it says "libc++abi: terminating due to uncaught exception of type NSException". 
I've sett the modpack to launch in fullscreen, I've changed the Java version to Amazon Corsetta JDK 8, and I've updates Forge to 14.23.5.2855 and Optifine to the G6 preview.

My device is a 14" M1 Pro Macbook Pro with 16GB of ram

**To Reproduce**
1. Launch modpack
2. Go through loading screen
3. Crash

**Expected behavior**
I expect the modpack to launch and run past the loading screen.

**Screenshots**

**Crash reports or logs**
There is no crash report generated; but there is a latest.log file here: https://paste.ee/p/zXSEm

**Details**
Please fill out the blanks below:
Modpack Version: (1.35.2.118.1)
Did you add or remove mods? (No)
Did you edit any config or script? (No)
Using shaders? [Write 'Default' if you are using default shader, if you are not using the default shader please write the name of the shader you are using.]  (Default)
Singleplayer, LAN game or server pack? (Singleplayer)

**Additional context**

**Comment by AR-E** [comment]
Here's the launcher log since I don't think the latest.log file has anything helplful: https://paste.ee/p/vqaZn

**Comment by AR-E** [comment]
I managed to figure it out. I changed 2 things, and now it launches:
1. I'm using Prism Launcher, and I changed the window width and height to match the resolution that Minecraft launches at 
2. I went to the options.txt and turned off fullscreen from there so that I could make it fullscreen using the maximize button in the title bar
I'm not sure which of these made it work (or if it was a combination of the 2), but I was able to launch the game properly.

**Comment by AR-E** [comment]
I managed to figure it out. I changed 2 things, and now it launches:
1. I'm using Prism Launcher, and I changed the window width and height to match the resolution that Minecraft launches at 
2. I went to the options.txt and turned off fullscreen from there so that I could make it fullscreen using the maximize button in the title bar

I'm not sure which of these made it work (or if it was a combination of the 2), but I was able to launch the game properly.
